elifearticle/article.py

```diff
@@ -421,6 +422,40 @@ def init(self):
         self.content_type = None
 
 
+class ClinicalTrial(object):
+
+    def __new__(cls):
+        new_instance = object.__new__(cls)
+        new_instance.init()
+        return new_instance
+
+    def init(self):
```

Comment:
a few things:

1. I don't think objects are necessary here. You have a single method and no ancestor. If it's just for consistency I can appreciate that, but objects are almost never necessary and should be avoided if possible. Feel free to ignore me on this.
2. shouldn't it be `__init__` ? and `__new__` seems to be compensating for the fact that there is no `__init__`

Reply:
I think this was a something I fixed elsewhere when realising the name wasn't `__init__`. The `__new__` was suggested by a linter long ago to use the new Python function, but I don't know if that's still a recommendation. I might open a new issue to rename all the `init()` ones.

Reply:
which linter are you using?

Reply:
I'd have to guess it was `pylint` three or four years ago when this project was in `py27`. The recommendations it gives may change based on today?

Reply:
it's just such a weird pattern, I don't think I've ever seen it or know why it would be recommended.

Reply:
The `init()` I have here should have been `__init__()`, it was an early typo I never noticed. I remember changing similar things in another repo (probably `elife-bot`) but I didn't change it here.